### DISCUSSION TITLE
Modal showing model data

### DISCUSSION THREAD

**Discussion by massimomarazzi** [filamentphp/filament#2298] [General]
Hi there, thank you for your awesome packages.
I use filament form and table.

I would need to make a button in table row that open a modal window with data from model, only showing data not edit.

How can i achive this results with filament? there is any method like for confirmation modal?
I want to avoid creating another modal window.

Thanks in advance.
Max

**Comment by danharrin** [comment]
You can now add a `ViewAction::make()` to your table. If you're using the tables package outside the admin panel resources, you'll need to pass a `->form()` with your disabled form components to display data.
